assets/chezmoi.io/docs/user-guide/password-managers/index.md

```diff
@@ -1,6 +1,30 @@
-# Password manager integration
+# Password Manager Integration in Chezmoi
```

Comment:
I would remove the `in Chezmoi`; it doesn't match any of the other titles in the user guide.